omf-python/src/file/reader.rs

```diff
@@ -400,7 +415,10 @@ impl PyReader {
             .map(|b| PyBytes::new_bound(py, &b))
     }
 
-    /// Read a Number array.
+    /// Read a Number array and return a tuple of two numpy arrays: the values
+    /// themselves, and a mask indicating any values that are null. The number
+    /// array’s type is one of :class:`numpy.float32`, :class:`numpy.float64`,
+    /// :class:`numpy.int64`, or :class:`numpy.datetime64`.
```

Comment:
I think this should document the exact dtypes that can be returned, which are IIRC `"datetime64[us]"` for timestamps and `"datetime64[D]"` for dates.